<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableWindow</class>
 <widget class="QMainWindow" name="tableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>520</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>411</width>
      <height>291</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>字符串</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>频率</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>呈现顺序：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combobox">
       <item>
        <property name="text">
         <string>降序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>升序</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
